Sources/Jobs/Scheduler/Scheduler.swift

```diff
@@ -0,0 +1,500 @@
+import Foundation
+
+protocol MonthBuilder {
```

Comment:
I'm not sure we need protocols here. Are there different types of schedulers we would need to conform to these? I think it could be done w/ concrete types, but i might be mistaken.

Reply:
Yeah I think it could probably be done with concrete types. I think you may have made the same comment on Reid's initial code but I forgot to make that change when I carried it over.